tests/unittests/graphTest.cpp

```diff
@@ -45,6 +45,57 @@ TEST(Graph, testVariableErasure) {
   EXPECT_EQ(std::distance(vars.begin(), vars.end()), vars.size());
 }
 
+/// Check that the clear method completely reset a module.
+TEST(Graph, clear) {
+  Module M;
+
+  // Check that the module is initially empty.
+  EXPECT_EQ(M.getVars().size(), 0);
```

Comment:
Is it worth our time to check that the module starts with zero vars, zero placeholders, zero functions and then check it again and again and again and again 7 times in the body of the function?

Reply:
We need to define a baseline and check that we get back to that base line when we call clear. I don’t expect this to be measurable chunk of time of our testing.

What is your concern here?

Reply:
This test is very long. I don't understand why we need to check that we didn't add new variables (and functions and placeholders) again and again.